pkg/controllers/status/rb_status_controller_test.go

```diff
@@ -219,3 +222,13 @@ func TestRBStatusController_syncBindingStatus(t *testing.T) {
 		})
 	}
 }
+
+var _ resourceinterpreter.ResourceInterpreter = &FakeResourceInterpreter{}
+
+type FakeResourceInterpreter struct {
+	*native.DefaultInterpreter
+}
+
+func (f FakeResourceInterpreter) Start(_ context.Context) (err error) {
+	return nil
+}
```

Comment:
In the future, `FakeResourceInterpreter` can be placed in `pkg/resourceinterpreter`.